docs/rules/README.md

```diff
@@ -8,266 +8,265 @@ Some rules are fixable using the `--fix` command line flag. Those rules are mark
 
 The following rules point out areas where you might have made mistakes.
 
-* [comma-dangle](comma-dangle.md): disallow or enforce trailing commas (recommended)
-* [no-cond-assign](no-cond-assign.md): disallow assignment in conditional expressions (recommended)
-* [no-console](no-console.md): disallow use of `console` (recommended)
-* [no-constant-condition](no-constant-condition.md): disallow use of constant expressions in conditions (recommended)
+* [comma-dangle](comma-dangle.md): require or disallow trailing commas (recommended)
+* [no-cond-assign](no-cond-assign.md): disallow assignment operators in conditional expressions (recommended)
+* [no-console](no-console.md): disallow the use of `console` (recommended)
+* [no-constant-condition](no-constant-condition.md): disallow constant expressions in conditions (recommended)
 * [no-control-regex](no-control-regex.md): disallow control characters in regular expressions (recommended)
-* [no-debugger](no-debugger.md): disallow use of `debugger` (recommended)
-* [no-dupe-args](no-dupe-args.md): disallow duplicate arguments in functions (recommended)
-* [no-dupe-keys](no-dupe-keys.md): disallow duplicate keys when creating object literals (recommended)
-* [no-duplicate-case](no-duplicate-case.md): disallow a duplicate case label. (recommended)
+* [no-debugger](no-debugger.md): disallow the use of `debugger` (recommended)
+* [no-dupe-args](no-dupe-args.md): disallow duplicate arguments in `function` definitions (recommended)
+* [no-dupe-keys](no-dupe-keys.md): disallow duplicate keys in object literals (recommended)
+* [no-duplicate-case](no-duplicate-case.md): disallow duplicate case labels (recommended)
 * [no-empty](no-empty.md): disallow empty block statements (recommended)
-* [no-empty-character-class](no-empty-character-class.md): disallow the use of empty character classes in regular expressions (recommended)
-* [no-ex-assign](no-ex-assign.md): disallow assigning to the exception in a `catch` block (recommended)
-* [no-extra-boolean-cast](no-extra-boolean-cast.md): disallow double-negation boolean casts in a boolean context (recommended)
+* [no-empty-character-class](no-empty-character-class.md): disallow empty character classes in regular expressions (recommended)
+* [no-ex-assign](no-ex-assign.md): disallow reassigning exceptions in `catch` clauses (recommended)
+* [no-extra-boolean-cast](no-extra-boolean-cast.md): disallow double-negation boolean casts (recommended)
```

Comment:
disallow unnecessary boolean casts

The original description had 2 issues:
* “boolean cast” refers to either !! or Boolean()
* “in a boolean context” was trying to communicate something similar to “unnecessary” in no-extra-parens